src/ring_theory/localization.lean

```diff
@@ -743,6 +744,89 @@ end away
 
 end away
 
+section inv_submonoid
+
+variables (M S) [is_localization M S]
+
+lemma image_invertible : M.map (algebra_map R S : R →* S) ≤ is_unit.submonoid S :=
+by { rintros _ ⟨a, ha, rfl⟩, exact is_localization.map_units S ⟨_, ha⟩ }
+
+/-- The submonoid of `S = M⁻¹R` consisting of `{ 1 / x | x ∈ M }`. -/
+@[nolint unused_arguments] -- This should only be defined when `S` is indeed the localization.
```

Comment:
Maybe we still define it in general? That also means that Lean needs to find fewer typeclasses just when using this defn.